hack/verify-chart-version.sh

```diff
@@ -20,7 +20,7 @@ set -o pipefail
 semvercompareOldVer=""
 semvercompareNewVer=""
 
-if [ "${PULL_BASE_SHA}" == "" ]; then
+if [ -z "${PULL_BASE_SHA+a}" ]; then
```

Comment:
What does the +a do here?

Reply:
It's a [parameter substitution](https://www.tldp.org/LDP/abs/html/parameter-substitution.html) that substitutes if a variable is set, and otherwise returns an empty string if the variable is unset.

That is to say:

```shell
x=1
# y is unset
${x+arbitrary string} # becomes "arbitrary string"
${y+arbitrary string} # becomes ""
```

It's a way to reference a variable to check if it's set without triggering an error from `set -o nounset`

Reply:
Thanks for the explanation and example! 😄